exercises/say/Say.elm

```diff
@@ -1 +1,11 @@
-module Say exposing (say, SayError(..))
+module Say exposing (SayError(..), say)
```

Comment:
Consider `SayError(Negative, TooLarge),` to make consistent with the `robot-simulator` exercise [line 3](https://github.com/exercism/elm/pull/207/files#diff-fadb708b3de456706ec84db3e97b01a6R3).

Reply:
I agree with `(..)` feeling like a code smell, in regards to modules. When it comes to exposing union types, I'm not sure `(..)` is actually a code smell. Reason is there are typically only 2 states the union is exposed in
1. Only the type and none of the constructors
2. Everything using `(..)`

Any other kind would include exposing some of the type constructors, which I don't think I've seen anyone do, despite being able to.

Reply:
That's a good point. 

In that case, maybe consider updating `robot-simulator` [line 3](https://github.com/exercism/elm/pull/207/files#diff-fadb708b3de456706ec84db3e97b01a6R3) to `( Bearing(..)`.

Reply:
Ah, I didn't even notice that before. I'll do that.